compiler/qsc_ast/src/ast.rs

```diff
@@ -317,14 +317,27 @@ impl Display for Attr {
 }
 
 /// A type definition.
-#[derive(Clone, Debug, PartialEq, Default)]
+#[derive(Clone, Debug, PartialEq)]
 pub struct TyDef {
     /// The node ID.
     pub id: NodeId,
     /// The span.
     pub span: Span,
     /// The type definition kind.
     pub kind: Box<TyDefKind>,
+    /// The documentation.
+    pub doc: Rc<str>,
+}
+
+impl Default for TyDef {
+    fn default() -> Self {
+        TyDef {
+            id: NodeId::default(),
+            span: Span::default(),
+            kind: Box::default(),
+            doc: Rc::from(""),
+        }
+    }
 }
 
 impl Display for TyDef {
```

Comment:
The display for `TyDef` should be updated here, which will make the tests actually show the parsed doc-comments.